### PR TITLE
added functionality to do multiple replaces

### LINKED ISSUE #306: Cannot stub more than 1 element?
When I tried to use replace() multiple times to stub out multiple elements, sinon complains:

> Error: Attempted to wrap instanceTemplate which is already wrapped
>     Object.wrapMethod at /components/sinon/sinon-1.12.1.js:1210
>           Object.stub at /components/sinon/sinon-1.12.1.js:3007
>           Object.with at web-component-tester/browser.js:1555
> 

Is replace() designed to be used more than once? Or is it only limited to only stub out 1 element?

### PR DESCRIPTION
**Pull request by e111077**
Fixes #306

### CODE REVIEW

**Comment by e111077** [comment]
passes the following crude tests:

https://github.com/Polymer/web-component-tester/tree/multiple-replace-with-tests

**Comment by justinfagnani** [comment]
Did you see https://github.com/Polymer/web-component-tester/pull/294 ?

**Comment by e111077** [comment]
Nope. Should I close the other issue?

File: browser.js [hunk 1]
```diff
@@ -1992,4 +1997,4 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 }());
-//# sourceMappingURL=browser.js.map
\ No newline at end of file
+//# sourceMappingURL=browser.js.map
```
Comment (cdata) [inline browser.js]: What's the story here? Was this automatically added by something? Is there a corresponding map that should be checked in?

**Comment by cdata** [comment]
> crude tests

...the tests we deserve.

**Comment by cdata** [comment]
@azakus Elliot wrote actual tests for our testing tool. That was really nice of him. Do you think you could give us guidance on how to add them as actual tests for this tool?

File: browser.js (hunk 1 above)
Comment (e111077) [inline browser.js]: Git kept saying it was missing a newline at the end of the file and it does have a newline :/

Comment (cdata) [inline browser.js]: Oh, sorry, I misread the diff as adding the whole sourcemap comment. SGTM!

**Comment by txsmith** [comment]
We ran into the same issue today and applied this fix locally with success. Happy to report it works!

File: browser/mocha/replace.js [hunk 2]
```diff
@@ -22,6 +22,10 @@ extendInterfaces('replace', function(context, teardown) {
         // implementation for later:
         var originalInstanceTemplate = Polymer.Base.instanceTemplate;
 
+        if (Polymer.Base.instanceTemplate.isSinonProxy) {
+          Polymer.Base.instanceTemplate.restore();
```
Comment (justinfagnani) [inline browser/mocha/replace.js]: Rather than restoring each time, just stub once inside the if statement.

Comment (e111077) [inline browser/mocha/replace.js]: Further investigation shows that Sinon's API does not seem to support replacing the wrapped spy. Also looking at their GH for this function, it seems as if replacing it manually would just repeat their already-written code. I recommend that we restore if it is already wrapped in a spy. (stay the same)

WDYT?

**Comment by e111077** [comment]
Continued on #316

**Pull request closed without merging**